.circleci/config.yml

```diff
@@ -31,12 +48,12 @@ workflows:
             branches:
               ignore: /.*/
             tags:
-              only: /^[0-9]{10}$/
+              only: /[0-9]+\.[0-9]+\.[0-9]+(\.[0-9]+)?/
```

Comment:
The frontend team has actually been sometimes publishing versions like `2.1.4a` but I'm fine ignoring that for now.

Reply:
We are not gonna do that, most probably